include/boost/redis/connection.hpp

```diff
@@ -380,23 +470,22 @@ class run_op {
    void operator()(
       Self& self,
       order_t order,
-      system::error_code ec0,
-      system::error_code ec1,
-      system::error_code ec2,
-      system::error_code ec3,
+      system::error_code setup_ec,
+      system::error_code health_check_ec,
+      system::error_code reader_ec,
       system::error_code)
```

Comment:
I believe the error omitted here is the `writer_ec`, we should perhaps not omit it for better readability If necessary we can ignore it with `boost::core::ignore_unused`. I am however wondering why it is not handled below.